Include/exprtk/exprtk.hpp

```diff
@@ -2393,6 +2393,8 @@ namespace exprtk
                case e_eof   : return ";";
                default      : return "UNKNOWN";
             }
+
+            return "UNKNOWN";
```

Comment:
Isn't this line redundant? 2394 will return anyway.

Reply:
@TLCFEM:   You are right it definitely is redundant, however some static analysers (**Coverity** and even **MSVC**) fail to see th default case is being handled in the switch and require the return value or will throw a diagnostics warning and when warnings are treated as errors` (eg: -Werror)` it will terminate the compilation.